README.md

```diff
@@ -12,6 +12,22 @@ NetworkManager DBUS API
 $ npm install --save nm-api
 ```
 
+## Use
+To start `nm-api` as an HTTP server (default port defined internally is `1337`):
+```javascript
+const nm = require(’nm-api’);
+
+const port = Number(process.env.NM_SERVICE_API_PORT) || 1338;
+
+nm.HttpServer(port)
```

Comment:
Can we follow the Node core style for creating servers here? i.e. `nm.createHttpServer()` which returns the server / app, which has a `.listen(port[, host[, onListening]])` method